src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/SetAzureVirtualNetworkGatewayVpnClientConfig.cs

```diff
@@ -19,11 +19,17 @@
 using System;
 using System.Collections.Generic;
 using System.Management.Automation;
+using System.Security;
+using Microsoft.Azure.Commands.Network.VirtualNetworkGateway;
+using Microsoft.WindowsAzure.Commands.Common;
 using MNM = Microsoft.Azure.Management.Network.Models;
 
 namespace Microsoft.Azure.Commands.Network
 {
-    [Cmdlet(VerbsCommon.Set, "AzureRmVirtualNetworkGatewayVpnClientConfig"), OutputType(typeof(PSVirtualNetworkGateway))]
+    [Cmdlet(VerbsCommon.Set, 
+        "AzureRmVirtualNetworkGatewayVpnClientConfig",
+         DefaultParameterSetName = VirtualNetworkGatewayParameterSets.Empty), 
+        OutputType(typeof(PSVirtualNetworkGateway))]
```

Comment:
This needs to support ShouldProcess.  SupportsShouldProcess=true here, and call ShouldProcess or ConfirmAction in cmdlet execution